src/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs

```diff
@@ -0,0 +1,936 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This entire file can be excluded on `netstandard2.0` if you plan on keeping the throw PNSE approach.

Reply:
I thought we _had_ to preserve the PNSE approach. Are you saying we have the option _not_ to keep it? If yes, how would that look like and how would it affect customers consuming the new package version?

Reply:
I did like the idea of keeping the behavior as untouched as possible (meaning: keeping the PNSE), even if the callstack will differ a bit from now on.

I'm curious about your answer to my question, but in the meanwhile I'll apply your suggestion to remove the generator file from netstandard2.0.

Reply:
Done.